ddtrace/tracer/log.go

```diff
@@ -99,7 +106,7 @@ func logStartup(t *tracer) {
 		GlobalService:         globalconfig.ServiceName(),
 		LambdaMode:            fmt.Sprintf("%t", t.config.logToStdout),
 		AgentFeatures:         t.config.features,
-		AppSec:                appsec.Status(),
+		AppSec:                appsecStatus,
```

Comment:
Can't this just be a boolean?
```suggestion
		AppSec:                appsec.Enabled(),
```
It would simplify things... Do we have any reason for it to be a string?

Reply:
Indeed, a bool value makes it simpler and good enough for debugging purposes.
Done in https://github.com/DataDog/dd-trace-go/pull/1042/commits/4f68efd71656021a57827d97dde5161a2e2a06e6